cassandra/io/asyncorereactor.py

```diff
@@ -259,7 +259,13 @@ def _run_loop(self):
                     break
             self._started = False
 
-        log.debug("Asyncore event loop ended")
+        try:
+            log.debug("Asyncore event loop ended")
+        except Exception:
+            # TODO: Remove when Python 2 support is removed
+            # PYTHON-1266. If our logger has disappeared, there's nothing we
+            # can do, so just log nothing.
+            pass            
```

Comment:
Given that this verbiage is already used for the existing logging try/catch in this method we're starting to get into some ugly duplication here.  Prolly worth pulling this out into a function, something like:

```python
def maybe_log_debug(*args, **kwargs)
    try:
        log.debug(*args, **kwargs)
    except Exception:
        # TODO: Remove when Python 2 support is removed
        # PYTHON-1266. If our logger has disappeared, there's nothing we
        # can do, so just log nothing.
        pass
```

and then call with appropriate params.

This is a relatively minor thing so I can prolly also just clean it up after this gets merged if you don't wanna muck with it @tlasica.  Either way is good with me!

Reply:
I was thinking about it, but I decided not to do it:
1. there are 2 occurences, my rule of thumb is 3
2. all are local in the same function

Reply:
No worries, I can take care of it.  Thanks for the fix @tlasica!